cpt.yaml

```diff
@@ -127,3 +119,27 @@ tools:
   - name: cpt_fasta_translate
     owner: cpt
     tool_panel_section_label: Annotation
+    
+  - name: cpt_protein_blast_grouping
+    owner: cpt
+    tool_panel_section_label: Annotation
+
+  - name: cpt_prophage_relatedness
+    owner: cpt
+    tool_panel_section_label: Annotation
+
+  - name: cpt_intersect_adj
+    owner: cpt
+    tool_panel_section_label: Annotation
+
+  - name: phageterm
+    owner: mmonot
+    tool_panel_section_label: Annotation
+
+  - name: phagetermvirome
+    owner: bioit_sciensano
+    tool_panel_section_label: Annotation
```

Comment:
This one also seems to be https://gitlab.pasteur.fr/vlegrand/ptv/-/tree/master

> The previous PhageTerm program (single phage analysis only) is still available at https://sourceforge.net/projects/phageterm/ (for versions <3.0.0)

But also no Galaxy wrapper in github/gitlab.

The wrapper in the TS looks ok, a few things can be improved.